js/explore/scatter_repoIssues.js

```diff
@@ -54,7 +54,7 @@ function draw_scatter_repoIssues(areaID) {
 			.attr('class', 'd3-tip')
 			.offset([-10, 0])
 			.html(function(d) {
-				var tipstring = "<sub>[ "+d.valueY+" Open - "+d.valueX+" Merged ]</sub><br>"
+				var tipstring = "<sub>[ "+d.valueY+" Open - "+d.valueX+" Closed ]</sub><br>"
```

Comment:
Strangely... Doesn't the graph on software.llnl.gov/explore already say "Closed" ? That's what I see right now (pre-merge)